specs/command-line-user-experience.md

```diff
@@ -194,8 +194,7 @@ tar cf - /opt/myapp \| lxc exec dakara:c2 -- tar xvf -    | Make a tarball of /o
     file pull <source> [<source>...] <target>
 
 **Description**
-Copies file to or from the container. Supports rewriting the uid/gid/mode. This
-is only allowed for containers that are currently running.
+Copies file to or from the container. Supports rewriting the uid/gid/mode.
```

Comment:
Nope, see 5a328b7620e35fb5831d7b8b819d6a2f531c5d53

Though reading the commit message again, it's wrong and should have been "of a stopped container" :)